webpack/scenes/Tasks/helpers.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { translate as __ } from 'foremanReact/common/I18n';
 import store from 'foremanReact/redux';
 import { urlBuilder } from 'foremanReact/common/urlHelpers';
-import { addToast } from 'foremanReact/components/ToastsList/slice.js';
+import { addToast } from 'foremanReact/components/ToastsList/slice';
```

Comment:
This (not fixed) line causes a bug in Katello 4.13, toasts fail for several REX tasks. Can we backport this one line to Katello 4.13? :) I can open a PR against the stable branch if you'd like :)

Reply:
Which error does it cause? if this causes an error wouldnt the other imports be a problem as well?

Reply:
The error is caused if the line stays as is: 
`import { addToast } from 'foremanReact/components/ToastsList/slice.js';`
Not sure what exactly is causing the problem but the `.js` file name seems to cause issues. So whenever a REX job is run on 4.13, an empty error toast is dispatched.

![image](https://github.com/user-attachments/assets/134c4555-3a86-41dc-b310-84886eba5f17)

Reply:
Good to know! Probably will be good to open a pr to backport it, but thats for the Katello people to approve :)

Reply:
Yes, please feel free to open a PR in KATELLO-4.13 :+1: